### DISCUSSION TITLE
Can whisper components be replaced individually?

### DISCUSSION THREAD

**Discussion by coastal45** [openai/whisper#448] [Q&A] [answered]
Sorry this is more of a python/pip question rather than specific to whisper, but anyway...

I have whisper installed on Linux by `pip install git+https://github.com/openai/whisper.git` and is working correctly.

Since the original install, there is an updated model.py that seems to have significant improvements.
But when I downloaded it, file size is 298k, but installed file is only 9.7K.
I tried substituting it but then not working.

**Comment by jongwook** [comment, marked as answer]
Can you try deleting the package and reinstalling by

```bash
pip uninstall whisper
pip install git+https://github.com/openai/whisper.git
```
